docs/interactions.rst

```diff
@@ -0,0 +1,123 @@
+:orphan:
+
+.. currentmodule:: nextcord
+
+.. _client_index:
+   
+Interaction Commands
+======================
+
+As discord have added interaction commands a feature with alot of possiblities and ideas we at nextcord have decided to add them to our fleet of features,
+
+This doc will explain the innerworkings and how to use interaction commmands.
+
+We suggest you learn how to make regular commands before looking through here.
+
+How To Make A Simple Interaction Command
+------------------------------------------
+
+This right here is a simple ping pong Command made with Nextcords slash feature.
+
+.. code-block:: python3
+
+    @bot.slash_command(name="ping")
+    async def ping(interaction):
+        await interaction.response.send_message("Pong!")
+        
+The way it works is that you use the slash_command function to interact with the DiscordAPI "name" is the name of your slash command, And guild_ids is used to limit the guilds that the slash command is available also useful for testing as slash commands can take up to an hour to register
```

Comment:
Again to make the Document easier to read and more formal, I suggest these changes.

"The way it works is that you use the slash_command function to interact with the DiscordAPI. "name" Is the name of your slash command, and guild_ids is used to limit the guilds that the slash command is available to. This is also useful for testing, as global slash commands can take up to an hour to register."